frontend/src/components/SearchImage/SearchImage.jsx

```diff
@@ -1,28 +1,49 @@
-import React, { useRef } from 'react'
-import './SearchImage.scss';
+import React, { useRef, useState, useEffect } from "react";
+import "./SearchImage.scss";
 
 const SearchImage = ({ onChange, setInputValue, value }) => {
-    const searchRef = useRef();
+  const searchRef = useRef();
+  const [isSearchBarFull, setSearchBarFull] = useState(true);
 
-    function handleSearchSubmit(e) {
-        e.preventDefault();
-        const searchTerm = searchRef.current.value;
-        onChange(searchTerm);
+  function handleSearchSubmit(e) {
+    e.preventDefault();
+    const searchTerm = searchRef.current.value;
+    onChange(searchTerm);
+  }
+
+  const isSticky = () => {
+    const scrollTop = window.scrollY;
+    if (scrollTop > 20) {
```

Comment:
hmm.. `20` seems like a magic number to me :) maybe we can consider putting it to the top of the page as a constant (example can be found on app.jsx), so it would be easier for future fellows to adjust it easier from one place.

Reply:
resolve

Reply:
did some improvements here